packages/@atjson/source-html/src/annotations/p.ts

```diff
@@ -8,6 +8,6 @@ export class Paragraph extends BlockAnnotation<GlobalAttributes> {
   static type = "p";
 
   get rank() {
-    return (super.rank * 3) / 2;
+    return 15;
```

Comment:
This mentions that it is generated automatically, where / how? Do I need to update anything beyond what I've already done to ensure this won't get wiped out in the future?

Reply:
it's generated by a script we run manually. I don't anticipate us needing to run it again anytime soon? it just scrapes the html tag spec to generate annotation classes and like...unless they add a bunch of new tags I think we'd probably do any updates by hand

Reply:
These are the lines to change FWIW:

https://github.com/CondeNast/atjson/blob/latest/packages/%40atjson/source-html/scripts/generate-annotations.ts#L216-L218